src/scripts/main.js

```diff
@@ -2,7 +2,31 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tr = document.querySelector('tr');
 
-// write your code here
+const putLibraryOnTable = () => {
+  for (let i = 0; i < people.length; i++) {
+    const row = document.createElement('tr');
+    const objectKeys = Object.keys(people[i]);
+
+    for (let j = 0; j < 6; j++) {
```

Comment:
Remove the inner for loop and directly create and append the required table cells for name, gender, born, died, age, and century.